libs/components/tabs/src/lib/modules/tabs/tabset.component.spec.ts

```diff
@@ -1273,14 +1277,24 @@ describe('Tabset component', () => {
       }
     }));
 
-    it('should have tabindex of 0', fakeAsync(() => {
+    it('should have tabindex of 0 when active', fakeAsync(() => {
       fixture.detectChanges();
       tick();
       fixture.detectChanges();
 
-      const butEl = debugElement.queryAll(By.css('.sky-btn-tab'))[1]
+      const butEl = debugElement.queryAll(By.css('.sky-btn-tab'))[0]
         .nativeElement;
       expect(butEl.getAttribute('tabindex')).toBe('0');
+    }));
+
+    it('should have tabindex of -1 when not active', fakeAsync(() => {
```

Comment:
tragically the hundreds of `fixture.detectChanges()` and `tick()`s in here are necessary--there are a lot of `setTimeouts` in tabset